inventory-fusion/src/test/resources/fusion-report/dsc-agent.ocs

```diff
@@ -0,0 +1,507 @@
+      <PNPDEVICEID>PCI\VEN_8086&amp;DEV_100E&amp;SUBSYS_001E8086&amp;REV_02\3&amp;267A616A&amp;0&amp;18</PNPDEVICEID>
+      <SPEED>1000</SPEED>
+      <STATUS>Up</STATUS>
+      <TYPE>ethernet</TYPE>
+      <VIRTUALDEV>0</VIRTUALDEV>
+    </NETWORKS>
+    <NETWORKS>
+      <DESCRIPTION>Intel(R) PRO/1000 MT Desktop Adapter</DESCRIPTION>
+      <IPADDRESS6>fe80::603e:5a81:67bd:dc36</IPADDRESS6>
+      <IPMASK6>ffff:ffff:ffff:ffff::</IPMASK6>
+      <IPSUBNET6>fe80::</IPSUBNET6>
+      <MACADDR>08:00:27:79:8A:76</MACADDR>
+      <PCIID>8086:100E:001E:8086</PCIID>
+      <PNPDEVICEID>PCI\VEN_8086&amp;DEV_100E&amp;SUBSYS_001E8086&amp;REV_02\3&amp;267A616A&amp;0&amp;18</PNPDEVICEID>
+      <SPEED>1000</SPEED>
+      <STATUS>Up</STATUS>
+      <TYPE>ethernet</TYPE>
+      <VIRTUALDEV>0</VIRTUALDEV>
+    </NETWORKS>
+    <NETWORKS>
+      <DESCRIPTION>Intel(R) PRO/1000 MT Desktop Adapter #2</DESCRIPTION>
+      <IPADDRESS>192.168.51.3</IPADDRESS>
+      <IPMASK>255.255.255.0</IPMASK>
+      <IPSUBNET>192.168.51.0</IPSUBNET>
+      <MACADDR>08:00:27:B2:4E:60</MACADDR>
+      <PCIID>8086:100E:001E:8086</PCIID>
+      <PNPDEVICEID>PCI\VEN_8086&amp;DEV_100E&amp;SUBSYS_001E8086&amp;REV_02\3&amp;267A616A&amp;0&amp;40</PNPDEVICEID>
+      <SPEED>1000</SPEED>
+      <STATUS>Up</STATUS>
+      <TYPE>ethernet</TYPE>
+      <VIRTUALDEV>0</VIRTUALDEV>
+    </NETWORKS>
+    <NETWORKS>
+      <DESCRIPTION>Intel(R) PRO/1000 MT Desktop Adapter #2</DESCRIPTION>
+      <IPADDRESS6>fe80::60e3:fe35:466a:aaad</IPADDRESS6>
+      <IPMASK6>ffff:ffff:ffff:ffff::</IPMASK6>
+      <IPSUBNET6>fe80::</IPSUBNET6>
+      <MACADDR>08:00:27:B2:4E:60</MACADDR>
+      <PCIID>8086:100E:001E:8086</PCIID>
+      <PNPDEVICEID>PCI\VEN_8086&amp;DEV_100E&amp;SUBSYS_001E8086&amp;REV_02\3&amp;267A616A&amp;0&amp;40</PNPDEVICEID>
+      <SPEED>1000</SPEED>
+      <STATUS>Up</STATUS>
+      <TYPE>ethernet</TYPE>
+      <VIRTUALDEV>0</VIRTUALDEV>
+    </NETWORKS>
+    <OPERATINGSYSTEM>
+      <ARCH>64-bit</ARCH>
+      <BOOT_TIME>2017-05-18 00:12:35</BOOT_TIME>
+      <DNS_DOMAIN></DNS_DOMAIN>
+      <FQDN>agent1</FQDN>
+      <FULL_NAME>Microsoft Windows Server 2008 R2 Standard </FULL_NAME>
+      <INSTALL_DATE>2014-10-02 15:07:08</INSTALL_DATE>
+      <KERNEL_NAME>MSWin32</KERNEL_NAME>
+      <KERNEL_VERSION>6.1.7600</KERNEL_VERSION>
+      <NAME>Windows</NAME>
+      <SERVICE_PACK />  <TIMEZONE>
+        <NAME>Pacific Standard Time</NAME>
+        <OFFSET>-0700</OFFSET>
+      </TIMEZONE>
+    </OPERATINGSYSTEM>
+    <PRINTERS>
+      <DRIVER>Microsoft XPS Document Writer</DRIVER>
+      <NAME>Microsoft XPS Document Writer</NAME>
+      <NETWORK>0</NETWORK>
+      <PORT>XPSPort:</PORT>
+      <PRINTPROCESSOR>winprint</PRINTPROCESSOR>
+      <RESOLUTION>600x600</RESOLUTION>
+      <SHARED>0</SHARED>
+      <STATUS>Idle</STATUS>
+    </PRINTERS>
+    <RUDDER>
+      <AGENT>
+        <AGENT_NAME>windows-dsc</AGENT_NAME>
+        <CFENGINE_KEY >-----BEGIN RSA PUBLIC KEY-----

```

Comment:
That has no impact of our inventory parsing but yes i added by hand to have my inventory accepted (and maybe we should change to AGENT_KEY